activesupport/lib/active_support/broadcast_logger.rb

```diff
@@ -229,6 +229,7 @@ def initialize_copy(other)
     private
       def dispatch(&block)
         @broadcasts.each { |logger| block.call(logger) }
+        nil
```

Comment:
Are you sure `Logger` returns `nil`? It says it returns `true` https://github.com/ruby/logger/blob/0996f90650fd95718f0ffe835b965de18654b71c/lib/logger.rb#L669

Reply:
I updated the pull request description to clarify that I'm referring to `ActiveSupport::Logger`.

Reply:
Let me know. I tested just with `ActiveSupport::Logger` inside the Rails framework and it always returns `true` like it is expected since it is just a subclass of `::Logger`.